examples/opt_sampling.ipynb

```diff
@@ -5,9 +5,9 @@
    "id": "42a35c1e-3424-4d93-ac13-ef3ff2869056",
    "metadata": {},
    "source": [
-    "In previous versions of imf, making a cluster using a mass function required sampling from it randomly. For a sufficient number of draws, the results can closely approximate the mass function. However, there is still remaining stochasticity, and different ways of sampling randomly may yield different results (or, at least, require multiple realizations to converge). In short, random sampling does not guarantee the ability to construct a cluster that fully utilizes a provided mass reservior to completely sample a provided mass function. That guarantee can only be made by using optimal sampling.\n",
```

Comment:
I removed this text because it is not true.  Random sampling from a mass function is not meant to approximate the mass function, it is meant to be a random draw from it.  The stochasticity is intrinsic to random sampling.  There is a fundamental physical and philosophical difference between random and optimal sampling - and we don't know which is more accurate!  (imo)